src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.*;
+
+public class Duke{
+    Scanner sc = new Scanner(System.in);
+    Task[] t = new Task[100];
+    private static int taskNo = 0;
+
+    public void run() {
+
+        String input;
+        String[] switchChoice;
+        do {
+            Ui.printMenu();
+            input = sc.nextLine().trim().toLowerCase();
```

Comment:
There is quite a few functions within the same line. Maybe we can split it to different lines to improve readability and understanding?